src/pybind/mgr/dashboard/ci/cephadm/bootstrap-cluster.sh

```diff
@@ -11,7 +11,7 @@ mkdir -p /etc/ceph
 mon_ip=$(ifconfig eth0  | grep 'inet ' | awk '{ print $2}')
 
 bootstrap_extra_options='--allow-fqdn-hostname --dashboard-password-noupdate'
-bootstrap_extra_options_not_expanded='--skip-monitoring-stack'
+bootstrap_extra_options_not_expanded=''
```

Comment:
Looks great! 
Can we instead remove all these lines (14-17)? As anyways we don't need extra options or we can just comment these lines maybe with a comment that they are temporarily commented, if any extra options needs to be added in future can be done here. Wdyt @nizamial09 ?

Reply:
Done! @avanthakkar